R/get_temperature.R

```diff
@@ -0,0 +1,107 @@
+#' @title Get temperature observations
+#' @description Temperature observations for different spatial resolutions.
+#'
+#' @details `get_temperature()` is a wrapper function for `obs_weather_monthly()`, `obs_weather_daily()`
+#' and `obs_weather_hourly()`. User specifies which spatial resolution (hourly, daily or monthly)
+#' they want the data in. At least one location parameter has to be given. The function can also
+#' label variables by using `label_variables()` and add info on station(s) by using `add_station()`.
+#'
+#'
+#' @param hourly A logical for whether to return hourly precipitation observations.
+#' @param daily A logical for whether to return daily precipitation observations.
+#' @param monthly A logical for whether to return monthly precipitation observations.
```

Comment:
I think it might be better if these 3 parameters were combined into 1 and input could be in the form of character (`"hourly"`) or a character vector (`c("hourly", "daily")`)

Reply:
Or, since "Function can only return data for one spatial resolution at the time", the parameters (could be named something like `interval` or sth) would accept only vector with length 1

Reply:
It makes sense to combine them into a single variable. Probably as you suggested that user can just provide character for the spatial resolution.